src/main/resources/data.sql

```diff
@@ -58,7 +58,7 @@ INSERT INTO "TASK" (ID, VERSION, TASK_CONTACT_ID, LINK, DESCRIPTION, DUE_DATE, U
 (400, 1, 11, 'https://www.linkedin.com/jobs/collections/recommended/?currentJobId=3773696405', 'C# Project', '2023-12-01', 'daniel'),
 (401, 1, 13, 'https://www.linkedin.com/jobs/view/3737405768/?alternateChannel=search&refId=kJ9G6Hld0CaMiRz3uur9jA%3D%3D&trackingId=FeyENqIJVtxTY1QL5UcF7g%3D%3D', 'Salesforce Opportunity with Nordea', '2024-01-05', 'laurids');
 INSERT INTO "USER_TABLE" (ID, FIRST_NAME, LAST_NAME, EMAIL, PASSWORD, VERSION) VALUES
-(500, 'Laruids', 'Lind', 'laurids.lind@consid.com', '{bcrypt}$2a$10$GRLdNijSQMUvl/au9ofL.eDwmoohzzS7.rmNSJZ.0FxO/BTk76klW', 1);
+(500, 'Laruids', 'Lind', 'laurids.lind@consid.com', '$2a$10$GRLdNijSQMUvl/au9ofL.eDwmoohzzS7.rmNSJZ.0FxO/BTk76klW', 1);
```

Comment:
I do not see any unit tests covering this. Please add unit tests before creating a new PR. :)